### PR TITLE
Added remaining qty and transferred qty on dialog box(Cancellation process of stock entry) 

### PR DESCRIPTION
**Pull request by viralkansodiya**
Screenshot 2024-03-09 185754

"As you can see in the dialog box in the above picture, that is the dialog box that pops up during the stock entry cancellation process.

I have added the current quantity of both handling units."

#91

### CODE REVIEW

File: beam/public/js/stock_entry_custom.js [hunk 1]
```diff
@@ -100,21 +115,32 @@ async function show_handling_unit_recombine_dialog(frm) {
 	})
```
Comment (agritheory) [inline beam/public/js/stock_entry_custom.js:115]: On line 111, we should change this to 'extra-large'. [Frappe example](https://github.com/frappe/frappe/blob/b331163ce96188f81cf7f5d8a67aa4aab9f92637/frappe/public/js/frappe/utils/diffview.js#L62)

**Review by agritheory — CHANGES_REQUESTED** [review]

File: beam/public/js/stock_entry_custom.js (hunk 1 above)
Comment (viralkansodiya) [reply beam/public/js/stock_entry_custom.js:115]: changes has been done

**Comment by viralkansodiya** [comment]
@agritheory 
any things pending in this PR?

**Review by agritheory — APPROVED** [review]

**Comment by agritheory** [comment]
@viralkansodiya Please run the linters again on this branch and push up any changes

**Pull request merged**